tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/AspNetCore5AsmToggle.cs

```diff
@@ -53,18 +53,14 @@ public AspNetCore5AsmToggle(AspNetCoreTestFixture fixture, ITestOutputHelper out
             SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "0");
         }
 
-        // TODO adjust third parameter as the following PRs are merged:
-        // * https://github.com/DataDog/dd-trace-dotnet/pull/3120
-        // * https://github.com/DataDog/dd-trace-dotnet/pull/3171
-        // the verify file names will need adjusting too
         [SkippableFact]
         [Trait("RunOnWindows", "True")]
         public async Task TestSecurityToggling()
         {
             uint expectedState = EnableSecurity == false ? ApplyStates.UNACKNOWLEDGED : ApplyStates.ACKNOWLEDGED;
             uint expectedCapabilities = EnableSecurity == false
-                                            ? (RcmCapabilitiesIndices.AsmIpBlockingUInt32 | RcmCapabilitiesIndices.AsmDdRulesUInt32)
-                                            : (RcmCapabilitiesIndices.AsmActivationUInt32 | RcmCapabilitiesIndices.AsmIpBlockingUInt32 | RcmCapabilitiesIndices.AsmDdRulesUInt32);
+                                            ? RcmCapabilitiesIndices.AsmIpBlockingUInt32 | RcmCapabilitiesIndices.AsmDdRulesUInt32 | RcmCapabilitiesIndices.AsmCustomBlockingResponseUInt32
```

Comment:
nit but could more readable written like this:
```
uint expectedCapabilities = RcmCapabilitiesIndices.AsmIpBlockingUInt32 | RcmCapabilitiesIndices.AsmDdRulesUInt32 | RcmCapabilitiesIndices.AsmCustomBlockingResponseUInt32;
if (EnableSecurity) {
    expectedCapabilities |= RcmCapabilitiesIndices.AsmActivationUInt32;
}
```

Reply:
thanks fixed in https://github.com/DataDog/dd-trace-dotnet/pull/3794/commits/109aa12c4886db146ae21e2652c8841105830022